hangman/lib/hangman/tally.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Hangman.Tally do
+  def tally(game = %Hangman.Struct{}) do
+    game
+    |> Map.delete(:word)
```

Comment:
I'd prefer to see a separate map which you copy values into.

Imagine you'd added a couple of secret fields to Hangman.Struct but then forgot to update this code. You'd then leak them to the outside.